platformics/codegen/generator.py

```diff
@@ -80,6 +80,10 @@ def generate_entity_subclass_files(
     template = environment.get_template(f"{template_filename}.j2")
 
     for entity in view.entities:
+        if entity.has_custom_policy:
```

Comment:
So I'm thinking that instead of using (even more) LinkML annotations, we might be able to solve this a different way.

Workflows already uses a template override configuration:
https://github.com/chanzuckerberg/czid-platformics/blob/main/workflows/Makefile#L96

In that case I think the Jinja2 template loader knows how to prioritize which template to use, but I'm wondering if the codegen script can (either through some Jinja2 config tweaks or just our own custom code) can see that there's a `cerbos/policies/bulk_downloads.yaml.j2` file in the overrides directory, and we're trying to output a file called `cerbos/policies/bulk_downloads.yaml` and automatically pick up the alternate template.

Reply:
Added a `entities/entities_templates/cerbos/policies/bulk_download.yaml.j2` template to override the generic template and updated `generator.py` to [use the alternate template](https://github.com/chanzuckerberg/czid-platformics/pull/279/files#diff-1d49981f58896a9348071b2aa889c179ce130fd953b7f72ff54ba5ded8ccf249R85) if it matches the output file name; lmk if you had something else in mind!

Reply:
This is great, thx!